opengever/apiclient/client.py

```diff
@@ -52,3 +52,6 @@ def get_navigation(self, raw=False):
                 'get_navigation currently does not support autowrapping its items, please use raw=True.'
             )
         return self.session().get(f'{self.url}/@navigation').json()
+
+    def update_object(self, **data):
+        return self.session().patch(self.url, json=data).ok
```

Comment:
AFAIK, the patch response contains the updated item, which could be very useful. In order to be consistent I'd return the new object, e.g.:
```python
    @autowrap
    def update_object(self, **data):
        return self.session().patch(self.url, json=data).json()
```

Reply:
The API returns no content: https://docs.onegovgever.ch/dev-manual/api/operations/#patch--(path)

Reply:
The GEVER-API returns "204 No Content" by default, see https://docs.onegovgever.ch/dev-manual/api/operations/#inhalte-bearbeiten-patch (I was not aware of that)
The apiclient should use the defaults of the GEVER-API, so the current implementation is the correct one.
When we need the object an another project, we will have to add a keyword argument and implement that later.